package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "private": true,
+  "repository": "https://github.com/TryGhost/Ghost-CLI",
+  "author": "Ghost Foundation",
+  "license": "MIT",
+  "workspaces": ["packages/*"],
+  "eslintignore": ["**/node_modules/**"],
+  "scripts": {
+    "presetup": "yarn",
+    "setup": "lerna bootstrap",
+    "test": "lerna run test",
+    "lint": "lerna run lint"
```

Comment:
Missing a ship command?

Reply:
yup, was gonna tackle that as part of the release workflow as well